client/my-sites/importer/site-importer/site-importer-input-pane.jsx

```diff
@@ -305,9 +309,15 @@ class SiteImporterInputPane extends React.Component {
 		const endpointParam =
 			this.state.selectedEndpoint && `?force_endpoint=${ this.state.selectedEndpoint }`;
 
+		const engineParam = this.props.importerData.engine
+			? `&engine=${ this.props.importerData.engine }`
```

Comment:
Wouldn't this produce malformed URL if `endpointParam` is `''` , i.e. `/site-importer/import-site&engine=foo`

Reply:
it is guaranteed to be after `?`: https://github.com/Automattic/wp-calypso/pull/28711/files#diff-4c5c89bd1d3d0cdd4e49e99c0890c64cR243